src/ui/components/spinner/Spinner.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { memo, useEffect, useRef } from 'react'
+import { Animated, Easing, Platform } from 'react-native'
+
+import { Logo } from 'ui/svg/icons/Logo'
+import { IconInterface } from 'ui/svg/icons/types'
+
+function NotMemoizedSpinner({ size, color }: IconInterface) {
```

Comment:
est-ce qu'on a vraiment besoin de ce spinner ? Si on peut l'enlever ça ferait pas de mal je pense. Encore une fois dans un soucis de simplicité et d'homogénéité. On l'a pas sur `SuggestedPlaces`. On est en train de builder un parcours assez compliqué, il faut aller à l'essentiel ⚠️

Reply:
Je vais le retirer alors ! 👍